aptos-move/framework/aptos-experimental/sources/trading/order_book/order_book_types.move

```diff
@@ -38,24 +38,26 @@ module aptos_experimental::order_book_types {
         idx: u128
     }
 
-    enum OrderType has store, drop, copy {
-        SingleOrder,
-        BulkOrder
+    struct OrderType has store, drop, copy {
+        // 0 - single order
```

Comment:
The issue with this change is that it is not flexible enough to extend to  support use case like MixedOrderType(u64, u64) in future.